src/TwbsHelper/Form/View/Helper/FormFile.php

```diff
@@ -14,31 +14,6 @@ class FormFile extends \Laminas\Form\View\Helper\FormFile
      */
     public function render(\Laminas\Form\ElementInterface $oElement): string
     {
-        $bCustom = $oElement->getOption('custom');
-
-        $sElementContent =  parent::render($this->setClassesToElement(
-            $oElement,
-            [$bCustom ? 'custom-file-input' : 'form-control-file'],
-            ['form-control']
-        ));
-
-        if ($bCustom) {
-            if ($sLabel = $oElement->getOption('custom_label')) {
-                $sLabelTmp = $oElement->getLabel();
-                $oElement->setLabel($sLabel);
-                $sLabel = $this->getView()->plugin('form_label')->__invoke($oElement);
-                $oElement->setLabel($sLabelTmp ?? '');
-                if ($sLabel) {
-                    $sElementContent .= PHP_EOL . $sLabel;
-                }
-            }
-
-            $sElementContent = $this->htmlElement(
-                'div',
-                ['class' => 'custom-file'],
-                $sElementContent
-            );
-        }
-        return $sElementContent;
+        return parent::render($oElement);
```

Comment:
If you reduce this class to this one method call, the entire class can be removed.
But the class for the HTML is now missing: [`form-control`](https://getbootstrap.com/docs/5.0/forms/form-control/#file-input) and also [the control for small or large](https://getbootstrap.com/docs/5.0/forms/form-control/#sizing).

Reply:
You are right, we can safely remove the entire class for file element view helper because there are no needed anymore.

To avoid class `form-control` missing we just need to remove `file` view helper from 'ignoredViewHelpers' array in config.
Custom classes for small or large is still available via element factory options (attributes -> class)

Reply:
> To avoid class form-control missing we just need to remove file view helper from 'ignoredViewHelpers' array in config.

That's the goal.